google/cloud/bigquery/schema.py

```diff
@@ -291,3 +327,96 @@ def _to_schema_fields(schema):
         field if isinstance(field, SchemaField) else SchemaField.from_api_repr(field)
         for field in schema
     ]
+
+
+class PolicyTagList(object):
+    """Define Policy Tags for a column.
+
+    Args:
+        names (Optional[List[str]]): list of policy tags to associate with
+            the column.  Policy tag identifiers are of the form
+            `projects/*/locations/*/taxonomies/*/policyTags/*`.
+    """
+
+    def __init__(self, names=None):
+        self._properties = {}
+        self.names = names
+
+    @property
+    def names(self):
+        """List[str]: Policy tags associated with this definition.
+        """
+        return self._properties.get("names", [])
+
+    @names.setter
+    def names(self, value):
+        """Optional[List[str]]: Policy tags associated with this definition.
+
+        (Defaults to :data:`None`).
+        """
+        if value is not None:
+            self._properties["names"] = value
+        else:
+            if "names" in self._properties:
+                del self._properties["names"]
+
+    def _key(self):
+        """A tuple key that uniquely describes this PolicyTagList.
+
+        Used to compute this instance's hashcode and evaluate equality.
+
+        Returns:
+            Tuple: The contents of this :class:`~google.cloud.bigquery.schema.PolicyTagList`.
+        """
+        return tuple(sorted(self._properties.items()))
+
+    def __eq__(self, other):
+        if not isinstance(other, PolicyTagList):
+            return NotImplemented
+        return self._key() == other._key()
+
+    def __ne__(self, other):
+        return not self == other
+
+    def __repr__(self):
+        return "PolicyTagList{}".format(self._key())
+
+    @classmethod
+    def from_api_repr(cls, api_repr):
+        """Return a :class:`PolicyTagList` object deserialized from a dict.
+
+        This method creates a new ``PolicyTagList`` instance that points to
+        the ``api_repr`` parameter as its internal properties dict. This means
+        that when a ``PolicyTagList`` instance is stored as a property of
+        another object, any changes made at the higher level will also appear
+        here.
+
+        Args:
+            api_repr (Mapping[str, str]):
+                The serialized representation of the PolicyTagList, such as
+                what is output by :meth:`to_api_repr`.
+
+        Returns:
+            google.cloud.bigquery.schema.PolicyTagList:
```

Comment:
(nit) Since this can now return `None`, the return type should be `Optional[google.cloud.bigquery.schema.PolicyTagList]`

Reply:
Done.